VIDEO_1_SCRIPT.md

```diff
@@ -55,23 +55,23 @@ E-commerce, Social media, cloud computing, mobility… Without a doubt, we are d
 
 Now that we have covered the origins and popularization of API, let's talk about the types of APIs.
 
-- ***Web services APIs***: these are the application development interfaces that enable the exchange of information between an application and a web service. (A web service is a software that provides access to a particular service via a URL). 
+* ***Web services APIs***: these are the application development interfaces that enable the exchange of information between an application and a web service. (A web service is a software that provides access to a particular service via a URL). 
 
     ***There are four types*** of Web services APIs:
 
-    •	***SOAP*** (Simple Object Access Protocol), ***a standard protocol for exchanging information and data in XML between two objects.***
+    *	***SOAP*** (Simple Object Access Protocol), ***a standard protocol for exchanging information and data in XML between two objects.***
 
-    •	***XML-RPC, a remote procedure call protocol that uses XML as the data format and HTTP calls as the communication system.***
+    *	***XML-RPC, a remote procedure call protocol that uses XML as the data format and HTTP calls as the communication system.***
 
-    •	***JSON-RPC, same as XML-RPC but in JSON format.***
+    *	***JSON-RPC, same as XML-RPC but in JSON format.***
 
-    •	***REST*** (Representational State Transfer), ***a software architecture for hypermedia systems on the World Wide Web; a REST API often uses the HTTP protocol.***
+    *	***REST*** (Representational State Transfer), ***a software architecture for hypermedia systems on the World Wide Web; a REST API often uses the HTTP protocol.***
 
 - ***Library-based APIs: These are APIs designed for a specific programming language. They are a means to reuse libraries from other software or other parts of the same software.***
 
 - ***Function APIs in operating systems: software programs are continuously interacting with operating systems (OS). In many cases, the way they do so is through APIs.*** Operating systems such as Windows have APIs that enable such communication between programs and the OS.
 
```

Comment:
```suggestion

We have just mentioned the Library and Function APIs, but we are going to dive into the Web APIs since they are the most relevant to AsyncAPI. 

```

How about saying this?

Reply:
sure, but I think we can skip `since they are the most relevant to AsyncAPI.`
- we do not have to explain ourselves 😄 
- first training videos are a general purpose, people do not need to understand what "AsyncAPI" means, so maybe we should not introduce that term

Reply:
Ok, ready!